main.go

```diff
@@ -120,11 +150,17 @@ func main() {
 		os.Exit(1)
 	}
 
-	metricsServer := osdmetrics.NewBuilder(config.OperatorNamespace, "localmetrics-"+config.OperatorName).
+	ns, err := k8sutil.GetOperatorNamespaceEnv()
```

Comment:
Can we re-use the namespace from the earlier call on line 117?

Reply:
Thank you Matt! Fixed it